adafruit_ble/characteristics/json.py

```diff
@@ -0,0 +1,59 @@
+# SPDX-FileCopyrightText: 2022 Mark Raleson
+#
+# SPDX-License-Identifier: MIT
+
+"""
+`json`
+====================================================
+
+This module provides Json characteristic.
+
+"""
+
+import json
+from . import Attribute
+from . import Characteristic
+
+__version__ = "0.0.0-auto.0"
+__repo__ = "https://github.com/adafruit/Adafruit_CircuitPython_BLE.git"
+
+
+class JsonCharacteristic(Characteristic):
```

Comment:
Let's use `JSON` instead of `Json`. The former is typical in existing Python libraries, including those that are included in CPython.
```suggestion
class JSONCharacteristic(Characteristic):
```